src/OpenCV/ImgProc/ImgFiltering.hsc

```diff
@@ -224,7 +225,7 @@ medianBlurImg = exceptError $
 -- TODO (Rvd): make ksize a type level argument
 -- if ksize in [3, 5] then depth in [Word8, Int16, Float) else depth ~ Word8
 medianBlur
-    :: ( depth    `In` '[Word8, Word16, Float]
+    :: ( depth    `In` '[Word8, Word16, Int16, Float, Double]
```

Comment:
`Int16` and `Double` are not allowed according to the documentation of [`medianBlur`](http://docs.opencv.org/3.1.0/d4/d86/group__imgproc__filter.html#ga564869aa33e58769b4469101aac458f9). Only `CV_8U`, `CV_16U`, or `CV_32F`.